app/views/home/index.html.erb

```diff
@@ -1,18 +1,39 @@
 <div class="jumbotron">
-    <h1 class="display-4">Connections Portal</h1>
-    <p class="lead">A digital tool for organizing and arranging networks, relationships and human connections.</p>
-    <hr>
-    <p>We hope you enjoy using it! <a href="https://github.com/hpi-swt2/connections-portal/issues/new" target="_blank">Give feedback!</a> </p>
-    <p class="lead" id="registration">
-        <%# fa_icon helper: https://github.com/bokmann/font-awesome-rails %>
-        <% if not user_signed_in? %>
-            <%= link_to (fa_icon 'sign-in', text: 'Log-In'), new_user_session_path, class: 'btn btn-primary btn-lg', id: "signin" %>
-            <%= link_to (fa_icon 'user-plus', text: 'Sign-Up'), new_user_registration_path, class: 'btn btn-secondary btn-lg', id: "signup" %>
-        <% end %>
-    </p>
-    <% if user_signed_in? %>
-        <hr>
-        <p>You are currently logged in as <%= current_user.email %>.</p>
-        <p>There are <%= Note.count %> stored notes.</p>
+  <h1 class="display-4">Connections Portal</h1>
+  <p class="lead">A digital tool for organizing and arranging networks, relationships and human connections.</p>
+  <hr>
+  <p>We hope you enjoy using it!
+    <a href="https://github.com/hpi-swt2/connections-portal/issues/new" target="_blank">Give feedback!</a></p>
+  <p class="lead" id="registration">
+    <%# fa_icon helper: https://github.com/bokmann/font-awesome-rails %>
+    <% if not user_signed_in? %>
+      <%= link_to (fa_icon 'sign-in', text: 'Log-In'), new_user_session_path, class: 'btn btn-primary btn-lg', id: "signin" %>
+      <%= link_to (fa_icon 'user-plus', text: 'Sign-Up'), new_user_registration_path, class: 'btn btn-secondary btn-lg', id: "signup" %>
     <% end %>
+  </p>
+  <% if user_signed_in? %>
+    <hr>
+    <p>You are currently logged in as <%= current_user.email %>.</p>
+    <p>There are <%= Note.count %> stored notes.</p>
+
+    <h2>Users</h2>
+    <table>
+      <thead>
+      <tr>
+        <th>Email</th>
+      </tr>
+      </thead>
+      <tbody>
+      <% User.all.each do |user| %>
+        <tr>
+          <td><%= user.email %></td>
```

Comment:
Ich würde hier vielleicht der +-Spalte eine Breite geben. Es sieht sehr gequetscht aus bisher.

Reply:
würden wir in einer zukünftigen UserStory schön machen... wurde so auch vom PO abgenommen